scripts/check.sh

```diff
@@ -26,9 +26,10 @@ check-protos-compatible() {
 
     if protolock status -lockdir=scripts -protoroot=proto; then
         protolock commit -lockdir=scripts -protoroot=proto
-        return 0
     else
-        return 1
+        echo "Meet break compatibility problem, please check the code."
```

Comment:
Is it possible to check out the latest version of proto.lock before all the local commits to compare with the current proto.lock to find compatibility issue? If we can't, the change looks good to me.

Reply:
After looking at the `protolock` source code we used, the code uses absolute paths and cannot set the path as github master. :(
And i add `git diff` check to avoid not adding `proto.lock` to git.

Reply:
How about only enabling it in the CI environment?

Reply:
If we put it in the ci, the development won't be able to commit the protolock every time when he makes a change.
Do you mean to make the robot implement add protolock Commit? I am not sure does it have this mechanism.